assets/html/package_details.html

```diff
@@ -5,6 +5,17 @@
 </head>
 <!-- The body is automatically populated by Expeditor with the marked down content from the release notes -->
 <!-- BEGIN AUTOGENERATED -->
-<body><p>This is a placeholder file. Eventually it will be used to tell customers information about the
-packages bundled with Chef Workstation.</p>
+<body><h1 id="chef-workstation-package-details">Chef Workstation Package Details</h1>
+<p>The following is a list of included software and their versions:</p>
+<ul>
+<li>Chef Workstation: 0.2.27<ul>
+<li>chef-run: 0.2.0</li>
+<li>chef-client: 14.5.33</li>
+<li>delivery-cli: master (6862f27aba89109a9630f0b6c6798efec56b4efe)</li>
+<li>berks: 7.0.6</li>
+<li>test-kitchen: 1.23.2</li>
+<li>inspec: 2.2.112</li>
```

Comment:
wont this end up being 3.0 once dk gets released?

Reply:
Yup! Good note, will need to update all these once ChefDK is built and I install a current channel build